prawcore/sessions.py

```diff
@@ -128,10 +164,12 @@ def _do_retry(
             method=method,
             params=params,
             url=url,
-            retries=retries - 1,
+            retry_strategy_state=retry_strategy_state.consume_available_retry(),
```

Comment:
This line is too long per flake8, but black forces it to be formatted this way. Ditto line 193.

Reply:
You can add `#noqa: e501` to the end of the line (I think that's the right thing to ignore).